Modules/Options/QuestieOptionsDefaults.lua

```diff
@@ -69,7 +69,8 @@ function QuestieOptionsDefaults:Load()
             trackerbindSetTomTom = 'ctrlleft',
             iconFadeLevel = 0.3,
             trackerLocked = true,
-            hideUnexploredMapIcons = false
+            hideUnexploredMapIcons = false,
+            mapRepeatableQuests = false,
```

Comment:
```suggestion
            showRepeatableQuests = true,
```

If we hide them by default users will be confused because it changes the current behavior!